lib/extract/compare_variables_mod.jinja

```diff
@@ -234,8 +236,13 @@ contains
     ! -------------------------------------------------------------------------
     !> @brief This subroutine writes a {{dim}}D array of {{type}}
```

Comment:
This bit of the docstring still talks about writing to file :-)

Reply:
Dang. Done, and I think I've reproduced files dependent on this as well.